oil_lang/run.sh

```diff
@@ -34,7 +34,7 @@ DISABLED-osh-cpp() {
 
   ninja $osh
```

Comment:
This function can be renamed to `test-parse-osh-cpp` or something, so then it will run in the CI without the DISABLED prefix

Reply:
We'll need to filter out `oil_lang/testdata/assign.osh` I think